clusters/hosted-mgmt/hive/pools/openstack-k8s-operators/openstack-k8s-operators-4-14-vexxhost-hybrid_clusterpool.yaml

```diff
@@ -20,12 +20,6 @@ spec:
   installAttemptsLimit: 3
   installConfigSecretTemplateRef:
     name: install-config-hybrid-3cp
-  inventory:
```

Comment:
You also need to update:
  runningCount: 5
  size: 5
  maxConcurrent: 5
  maxSize: 5

to zero :D

Reply:
too much rush to finish :D

Reply:
Done :)